rbac/management/migrations/0034_remove_field_uniqueness.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 2.2.4 on 2021-04-23 13:08
```

Comment:
I think we probably do still want to remove these existing constraints, but I'm just curious if the serializer validation fix you made in here solves the issue we were hitting with this? Just wondered if it was a red herring initially?

Reply:
I went through a few iterations on how best to handle this, but overall I do think this solves the issue we were initially hitting. It also moves the burden from enforcing uniqueness to the DB rather than the serializer, which seems like it should focus more on data validation than enforcing uniqueness rules.

Reply:
Sounds good!